packages/components/src/Components/FormElements/InputText/InputText.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import { formColors } from '../../../Utils/colors';
+
+const styles = {
+  text: css`
+    border: 1px solid ${formColors.colorInputBorder};
+    background: ${formColors.colorInputBg};
+    color: ${formColors.colorInputText};
+    border-radius: 2px;
+    padding: 1em;
+    font-size: 16px;
+    box-shadow: inset 0 0 0 1px transparent;
+    box-sizing: border-box;
+    &:placeholder {
+      color: ${formColors.colorInputPlaceholder};
+    }
+    &:hover {
+      border-color: ${formColors.colorInputFg};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputFg};
+    }
+    &:focus {
+      border-color: ${formColors.colorInputBorderFocus};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderFocus};
+      outline: none;
+    }
+  `,
+  textError: css`
+    border-color: ${formColors.colorInputBorderError};
+    box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderError};
+  `,
+  textDisabled: css`
+    color: ${formColors.colorInputBorderError};
+    background-color: ${formColors.colorInputBgDisabled};
+    &:hover {
+      border-color: ${formColors.colorInputBorder};
+      box-shadow: inset 0 0 0 1px transparent;
+    }
+  `,
+};
+
+/**
+ * Renders an Input type text.
+ *
+ * Example:
+ *
+ * @code
+ * <InputText type="text""></InputText>
+ * @endcode
+ */
+const InputText = props => {
+  const { error, disabled, placeholder } = props;
+  const cbClasses = [styles.text];
+  if (error) {
+    cbClasses.push(styles.textError);
+  }
+  if (disabled) {
+    cbClasses.push(styles.textDisabled);
+  }
+
+  return (
+    <input
+      type="text"
+      id={`${props.fieldName}-cb`}
+      name={props.fieldName}
+      value={
+        (Array.isArray(props.value) && props.value.length && props.value[0]) ||
```

Comment:
May I ask the usecase for this? Maybe we could add a comment when you want to support an array as value? It feels like something an outer component should handle